Model/Payment/Customer.php

```diff
@@ -96,6 +109,25 @@ public function createCustomer($body)
         return false;
     }
 
+
+    /**
+     * Update customer Vindi.
+     *
+     * @param string $query
+     *
+     * @return array|bool|mixed
+     */
+    public function updateVindiCustomer($query ,$body)
+    {
+        $response = $this->api->request("customers/{$query}", 'PUT', $body);
```

Comment:
Apenas para deixar mais explícito que esse parâmetro é um Id e não uma query, o que acha de alterar para algo como: 
```suggestion
    public function updateVindiCustomer($customerId ,$body)
    {
        $response = $this->api->request("customers/{$customerId}", 'PUT', $body);
```